exercises/K-functions-parameters/exercise2.js

```diff
@@ -1,7 +1,16 @@
-function divide(num1, num2){
-    return num1/num2;
-}// Declare your function first
+// function divide(num1, num2){
+//     return num1/num2;
+// }// Declare your function first
 
-var result = divide(3, 4);
+// var result = divide(3, 4);
 
-console.log(result);
+// console.log(result);
+
+// My copy.
+function divide(num1, num2) {
+    return num1 / num2;
+}
+
+let answer = divide(5, 10);
+
+console.log(Math.round(answer));
```

Comment:
Douglas the expected results show 0.75 in the instructions and I see you using Math.round?

Reply:
Thank you Jontaz77 for your comment. I'll review it and make the necessary changes.